packages/slider/src/Slider.ts

```diff
@@ -143,51 +165,39 @@ export class Slider extends Focusable {
         `;
     }
 
-    protected updated(changedProperties: PropertyValues): void {
-        if (changedProperties.has('value')) {
-            this.dispatchInputEvent();
-        }
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.handleController.hostConnected();
+        this.addEventListener(
+            'sp-slider-handle-update',
+            this.handleSliderHandleUpdate
+        );
     }
 
+    public disconnectedCallback(): void {
+        super.disconnectedCallback();
+        this.handleController.hostDisconnected();
+        this.removeEventListener(
+            'sp-slider-handle-update',
+            this.handleSliderHandleUpdate
+        );
+    }
+
+    private handleSliderHandleUpdate = (): void => {
+        this.requestUpdate();
+    };
+
     private renderLabel(): TemplateResult {
         return html`
             <div id="labelContainer">
-                <label id="label" for="input"><slot>${this.label}</slot></label>
+                <label id="label" for="input">${this.label}</label>
```

Comment:
I'd love to find an acceptable way to allow use to keep this `<slot>`. In the combobox research, we've looked at having the `sp-combobox-item` elements sprout a `slot='option'` attribute to allow for this. Thoughts on `<sp-slider-handle>` doing similar? Maybe sprouting `slot="handle"`? This is also a way to support the `assignedElements` note above.

Reply:
Ok, giving that a try.